wire-schema/src/test/java/com/squareup/wire/schema/internal/parser/ProtoParserTest.kt

```diff
@@ -512,7 +512,7 @@ class ProtoParserTest {
       ProtoParser.parse(location, proto)
     } catch (e: IllegalStateException) {
       assertThat(e).hasMessage(
-          "Syntax error in file.proto at 2:12: expected '//' or '/*'"
+          "Syntax error in file.proto at 2:13: expected '//' or '/*'"
```

Comment:
Bug?

Reply:
When I converted it from an `if` to a lambda I started sampling the location earlier. That impacts this one callsite.
```
    expect(pos < data.size && (data[pos] == '/' || data[pos] == '*')) {
      pos-- // Backtrack to start of comment.
      "expected '//' or '/*'"
    }
```
I didn’t want to make things more complex to support it, so I changed the test. Both offsets are okay!

Reply:
Sweet! Thought so but double-checked.